README.md

```diff
@@ -1,2 +1,12 @@
-# HW01
-Edit the readme
+# Idongesit Ekpo's Bio for HW 1
+Hello, my name is Idongesit Ekpo and I have lived in Utah for the past **four years**. Before living in Utah, I lived in FCT Abuja, Imo and Lagos. These are all Nigerian territories and states. Living in these places has shaped my life and how I interact with people. One life lesson that I learnt from all these places is that ***everyone has a story.*** Some of my hobbies are listed below. They include:
```

Comment:
Out of all the places you've lived, is there a place that you miss more than the others?

Reply:
I miss Abuja the most because that's where I was raised. :)